examples/esp32/src/bin/ble_bas_peripheral_sec.rs

```diff
@@ -17,7 +20,7 @@ async fn main(_s: Spawner) {
 
     let mut rng = esp_hal::rng::Trng::new(peripherals.RNG, peripherals.ADC1);
```

Comment:
I'm wondering what's the difference between 

```rust
esp_hal::rng::Rng::new(peripherals.RNG)
```

and 

```rust
esp_hal::rng::Trng::new(peripherals.RNG, peripherals.ADC1)
```
?

I found in different files, the `rng` initialization is different

Reply:
Thanks!